include/aikido/control/ros/RosBarrettHandPositionCommandExecutor.hpp

```diff
@@ -0,0 +1,81 @@
+// Assume it has 4 positions
+#ifndef AIKIDO_CONTROL_ROS_ROSBARRETTHANDPOSITIONCOMMANDEXECUTOR_HPP_
+#define AIKIDO_CONTROL_ROS_ROSBARRETTHANDPOSITIONCOMMANDEXECUTOR_HPP_
+#include <chrono>
+#include <future>
+#include <mutex>
+#include <ros/ros.h>
+#include <ros/callback_queue.h>
+#include <sensor_msgs/JointState.h>
+#include <pr_control_msgs/SetPositionAction.h>
+#include <actionlib/client/action_client.h>
+#include <Eigen/Dense>
+
+namespace aikido{
+namespace control {
+namespace ros {
+
+/// This class sends commands for the hand position to the ROS server
+class RosBarrettHandPositionCommandExecutor
```

Comment:
- This should inherit from `PositionCommandExecutor ` that is added in #166.
- This class is **not** specific to the BarrettHand, so it needs to be renamed. I suggest `RosPositionCommandExecutor` to mimic `RosTrajectoryExecutor`.